src/ServicePulse.Host/vue/vite.config.ts

```diff
@@ -69,7 +72,7 @@ export default defineConfig({
   },
   server: {
     headers: {
-      "Content-Security-Policy": createCSPOverrides(port, defaultUrls),
+      "content-security-policy": createCSPOverrides(port, defaultUrls),
```

Comment:
Headers are case insensitive, why this change?

Reply:
the [key in the dictionary is lowercase](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f5925b6c95f53653f35ce3f83fcda1ba9b1e04a6/types/node/http.d.ts#L142). It was being highlighted in vscode

<img width="1047" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/91dcec82-edce-4ba4-831b-009da12a08fb">

Reply:
I guess we can make it lowercase, but not required from the http specs